app/views/courses/qualification/_qts.html.erb

```diff
@@ -3,9 +3,6 @@
     Qualified teacher status (QTS) allows you to teach in state schools in England and may also allow you to teach in other parts of the UK.
   </p>
   <p class="govuk-body">
-    It may also allow you to teach in <%= govuk_link_to 'the EU and EEA', 'https://www.gov.uk/eu-eea' %>, though this could change after 2020.
-  </p>
-  <p class="govuk-body">
-    If you&rsquo;re planning to teach overseas, you should always check what qualifications are needed in the country you&rsquo;d like to teach in.
+    It may also allow you to teach overseas, though you should always check what qualifications are needed in the country you&rsquo;d like to teach in.
```

Comment:
```suggestion
    It may also allow you to teach overseas, though you should always check what qualifications are needed in the country you’d like to teach in.
```

Any reason not to do this?

Reply:
Nope. I just copied and pasted from the other files... 😄